crates/modelardb_server/src/optimizer/model_simple_aggregates.rs

```diff
@@ -750,23 +896,124 @@ impl Accumulator for ModelAvgAccumulator {
         Ok(())
     }
 
+    /// Panics as this method should never be called.
     fn merge_batch(&mut self, _states: &[ArrayRef]) -> Result<()> {
         unreachable!()
     }
 
+    /// Return the current state of the [`Accumulator`].
     fn state(&self) -> Result<Vec<ScalarValue>> {
-        //Must match datafusion::physical_plan::expressions::AvgAccumulator
+        // Match AvgAccumulator in repository/datafusion/physical-expr/src/aggregate/average.rs.
```

Comment:
Maybe add this as an actual link in the doc comment for the function.

Reply:
Fixed.